ec_grasp_planner/src/grasp_success_estimator.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+
+import yaml
+import rospy
+import tf
+import tf.transformations
+from enum import Enum
+
+# from std_msgs.msg import Int8 <-- Not used anymore since ROSTopicSensor only supports Float64
+from std_msgs.msg import Float64
+from std_msgs.msg import MultiArrayDimension
+from std_msgs.msg import Float64MultiArray
+from geometry_msgs.msg import WrenchStamped
+from hybrid_automaton_msgs.msg import HAMState
+
+from scipy.stats import norm
+
+import sys
+
+
+class RESPONSES(Enum):
+    REFERENCE_MEASUREMENT_SUCCESS = 0.0
+    REFERENCE_MEASUREMENT_FAILURE = 1.0
+
+    ESTIMATION_RESULT_NO_OBJECT = 10.0
+    ESTIMATION_RESULT_OKAY = 11.0
+    ESTIMATION_RESULT_TOO_MANY = 12.0
+    ESTIMATION_RESULT_UNKNOWN_FAILURE = 13.0
+
+    GRASP_SUCCESS_ESTIMATOR_INACTIVE = 99.0
+
+
+class ObjectModel(object):
+    def __init__(self, name, mass_mean, mass_stddev):
+        # The name of the object class (e.g. cucumber)
+        self.name = name
+        # mean of the object masses (e.g. over all cucumbers) in kg
+        self.mass_mean = mass_mean
+        # standard deviation of mass of the object (e.g. over all cucumbers)
+        self.mass_stddev = mass_stddev
+
+
+class MassEstimator(object):
+
+    # If confidence is lower than this threshold the estimator warns that confidence is low (will still report a result)
+    CONFIDENCE_THRESHOLD = 0.3
+
+    # FT-Topic to use. Possible options are: /ft_notool and /ft_sensor_wrench
+    FT_TOPIC = "/ft_notool"
```

Comment:
Since 822d33d this can be set via two arguments to the program. They are documented in the respective launch file in soma_utils.
This information should also be included into the README